webassembly-observe-sdk/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "30eb706f-9143-461e-99af-89015e8493d5",
+  "app_id": "webassembly-observe-sdk",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Extract traces, metrics, and logs from WebAssembly (wasm) code.",
+    "title": "WebAssembly Observe SDK",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Visualize captured logs, metrics, and traces from WebAssembly code running in your application.",
+        "image_url": "images/wasm-observability.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Get the open source Datadog adapter from the Dylibso Observe SDK via GitHub at https://github.com/dylibso/observe-sdk.",
+        "image_url": "images/wasm-observability-sdk.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Developer Tools",
+      "Category::Languages",
+      "Category::Metrics",
+      "Category::Log Collection",
```

Comment:
Are you sending in logs to Datadog?

Reply:
Yes, though to clarify, these are logs attached to spans within a trace. I can see that those could be different than other system/application logs though. Happy to remove this category if it isn't accurate.